package.json

```diff
@@ -16,10 +16,19 @@
         "stylelint-config-recommended": "^3.0.0"
     },
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.8.2",
```

Comment:
Fortawesome or fontawesome?

Reply:
FortAwesome is the organization: https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers#installing-free

Reply:
Oh I was not aware of that.